test/test_AlphaFilter.cpp

```diff
@@ -39,37 +45,36 @@
 
 using matrix::Vector3f;
 
-class AlphaFilterTest : public ::testing::Test {
```

Comment:
What was the reason for getting rid of the test fixture? Now it needs to be instantiated in each test instead of once at the top.

Reply:
Because I confused the two instances once and screwed up by setting the parameters on the wrong one and it silently passed. So I thought why not make it extra clear since they are completely exclusive to single tests. Having two fixtures to have two template instances seemed overkill.